sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S3329.json

```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Cipher Block Chaining IV's should be unpredictable",
+  "title": "Cipher Block Chaining IV\u0027s should be unpredictable",
```

Comment:
This title is actually interesting in that:
1. it's using a different char for `'`
2. It shouldn't be using `'` at all, as the `s` in `IVs` here is plural and not possessive. 
See https://github.com/SonarSource/rspec/pull/246